llama_stack/distribution/stack.py

```diff
@@ -105,3 +181,17 @@ async def construct_stack(
     )
     await register_resources(run_config, impls)
     return impls
+
+
+async def get_stack_run_config_from_template(template: str) -> StackRunConfig:
```

Comment:
does not need to be async. you don't expect this to be doing anything deeply asynchronous

Reply:
done.